src/Camunda8/Services/ConfigurationService.cs

```diff
@@ -4,23 +4,16 @@ public class ConfigurationService
 {
     // private readonly IConfiguration _configuration;
     public Camunda8Configuration Configuration { get; set; }
```

Comment:
```suggestion
    public Camunda8Configuration Configuration { get; private set; }
```

out of scope of this PR, but I'm just wondering -- do you want the `Configuration` property to have a public setter here? That would allow people to build/merge a configuration on their own...which you may or may not want.

Reply:
That's what I had in mind, so it makes sense to make it read-only.